Esri/arcgis-maps-sdk-kotlin-samples
# Show coordinates in multiple formats

## Description
PR to add a new Jetpack Compose sample _"Show coordinates in multiple formats"_ in `Edit and Manage Data` category.

## Links and Data
Sample Epic: `runtime/kotlin/issues/2571`

## What To Review
- Compose files and ViewModel files

## How to Test
- Run the app to test the coordinate updates on map tapped or updating the text fields. 
- I would suggest to tap the map at a location to obtain a coordinate. You can copy this coordinate and tap on a different location. Updating the text field using your copied coordinate should update the map graphic to the previews point.
